crates/bevy_render/src/camera/camera.rs

```diff
@@ -43,14 +58,21 @@ impl Default for DepthCalculation {
 }
 
 impl Camera {
+    pub fn get_window(&self) -> Option<WindowId> {
```

Comment:
As far as I can tell almost every caller of `get_window` doesn't care about the window, it just wants to know the size of the camera target.
What if we expose `fn get_size(&self, windows: &Windows, images: &Assets<Image>) -> Option<UVec2>` instead?

That way the usages are forced to support render-to-texture use cases as well instead of going the easy way of `.get_window().unwrap()`.

Reply:
Good call - I implemented exactly this (same name & signature!) before seeing your comment. There's still one place `get_window` is needed in `clear_pass.rs`, but I think it will go away eventually.